.env.example

```diff
@@ -1,7 +1,7 @@
-DB_USERNAME=postgres
-DB_PASSWORD=db_password
-DB_NAME=barefoot_nomad
-DB_HOST=127.0.0.1
-PORT=
-
-DB_TEST_NAME=barefoot_test
+DB_USERNAME=database username
+DB_PASSWORD=database password
+DB_NAME=name of database for development
+DB_HOST=Host url
+PORT=port number to listen for connections
+DB_TEST_NAME=name of test database
+SECRET=secret variable to sign and decode tokens
```

Comment:
Be more descriptive about the name `SECRET`  There are many secret key pair values in others people branch. Something like `SECRET_USER_ROLES_TOKEN`

Reply:
Thanks for the comment. That was just to get tokens. Creating of token is decided from the sign in route. That variable is a temporary one, pending when sign in route will be implemented.